### ISSUE TITLE
Skinning in party on quests that need to kill beasts

### ISSUE THREAD

**Issue by andreiacostinoaei** [dalaranwow/dalaran-wow#5323]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
While in a part group making quests together the beasts you have to loot that have quests are not skinnable after everyone took all the loot from the beast... For example 2 players in group making the quests from barrens where you have to loot 12 quest items from the raptors. If I loot the quest item the other player doesn't have quest item since I looted first and all loot on it. The second player will have some personal gray white loot and no quests item. Everything fine by now so the beast should be skinnable but it says it still has some loot on it but it is empty. So you can't skin it. But killing beasts in part that do not have quest items it works properly.
**Expected behaviour**: Tell us what should happen instead
Player 1 loots everything player 2 loots his personal loot. Player one should get most of the loot plus quest item. Player 2 should get some white gray items or nothing. The player with skinning should be able to skin the beast.
**Steps to reproduce the problem**:

1. Make quest with looting item from beast in a party group. Two players are enough.
2. Kill the beast and both players should loot everything.
3. The player with skinning try skin the beast. It will not work.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
